gitless/cli/gl_init.py

```diff
@@ -25,14 +25,21 @@ def parser(subparsers, _):
       help=(
           'an optional remote repo address from where to read to create the '
           'local repo'))
+  init_parser.add_argument(
+      '-o', '--only', nargs='*',
+      help='use only files given (tracked modified or untracked)', dest='only')
+  init_parser.add_argument(
+      '-e', '--exclude', nargs='+',
+      help = 'use everything but these branches', dest='exclude')
+
   init_parser.set_defaults(func=main)
 
 
 def main(args, repo):
   if repo:
     pprint.err('You are already in a Gitless repository')
     return False
-  core.init_repository(url=args.repo)
+  core.init_repository(url=args.repo, exclude=frozenset(args.exclude if args.exclude else []), only=frozenset(args.only if args.only else []))
```

Comment:
break up this line if it goes over 80 columns and same for the other lines you added. You can look at other parts of the code to see how we do it